core/templates/domain/utilities/FileDownloadRequestObjectFactory.ts

```diff
@@ -24,8 +24,10 @@ export class FileDownloadRequest {
   // TODO(YashJipkate): Replace 'any' with the exact type. This has kept as
```

Comment:
Could you please create an issue for this? I guess this will be removed after Angular migration? Ditto below

Reply:
filed #9996

Reply:
Please replace `YashJipkate` with the `#9996`. Ditto in other places.

Reply:
done